src/epd2in9b/mod.rs

```diff
@@ -0,0 +1,322 @@
+//! A simple Driver for the Waveshare 2.9" (B) E-Ink Display via SPI
+
+use embedded_hal::{
+    blocking::{delay::*, spi::Write},
+    digital::v2::*,
+};
+
+use crate::interface::DisplayInterface;
+use crate::traits::{
+    InternalWiAdditions, RefreshLUT, WaveshareDisplay, WaveshareThreeColorDisplay,
+};
+
+pub const WIDTH: u32 = 128;
+pub const HEIGHT: u32 = 296;
+pub const NUM_DISPLAY_BITS: u32 = WIDTH * HEIGHT / 8;
+pub const DEFAULT_BACKGROUND_COLOR: Color = Color::White;
+
+const IS_BUSY_LOW: bool = true;
+const VCOM_DATA_INTERVAL: u8 = 0x07;
+const WHITE_BORDER: u8 = 0x70;
+const BLACK_BORDER: u8 = 0x30;
+const RED_BORDER: u8 = 0xb0;
+const FLOATING_BORDER: u8 = 0xF0;
+
+use crate::color::{Color, TriColor};
+
+pub(crate) mod command;
+use self::command::Command;
+
+#[cfg(feature = "graphics")]
+mod graphics;
+#[cfg(feature = "graphics")]
+pub use self::graphics::Display2in9b;
+
+
+/// EPD2in9b driver
```

Comment:
Could you please add a simple example here?

Reply:
done!